app/buckets/handlers.js

```diff
@@ -49,25 +49,27 @@ exports.create_bucket = (req, res) => {
 
 
 exports.bucket_details = (req, res, next) => {
-  let bucket;
-  let users;
   Promise.all([Bucket.get(req.params.id), User.list()]) // need to include App.list() in future
-    .then(([returned_bucket, returned_users]) => {
-      bucket = returned_bucket;
-      users = returned_users;
-    })
-    .then(() => bucket.access_logs)
-    .then((access_logs) => { // need to include apps in future
+    .then(([bucket, users]) => { // need to include apps in future
```

Comment:
Is this comment necessary/true/relevant any more?

Reply:
It's still true and relevant. When webapps gets opened up to non-superusers this will need to include `apps` and a filtered version called `apps_options`.

Whether it's necessary is debatable, but the reason for having it hasn't changed or gone away.

Reply:
Ok, then